### PR TITLE
Feature/local environment

### PR DESCRIPTION
**Pull request by bastiankoetsier**
## What does this PR do?
Setup a local docker-environment for the gateway. It includes the /users endpoint of the auth-service as a fixture. 
Gateway is accessible through port 8888, the auth service itself through 8887. The datastores are exposed through their own ports for debugging purposes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,3 +36,12 @@ To start contributing, please check [CONTRIBUTING](CONTRIBUTING.md).
 
 ## Documentation
 * Go lang: https://golang.org/
+
+## Local development
```
Comment (wellfreire) [inline README.md:40]: +1 for adding documentation

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            context: .
+        volumes:
+            - ./:/go/src/github.com/hellofresh/janus
+        environment:
+            PORT: 3001
+            DEBUG: "true"
+            DATABASE_DSN: "mongodb://mongo:27017"
+            REDIS_DSN: "redis:6379"
+        ports:
+            - "8888:3000"
```
Comment (wellfreire) [inline docker-compose.yml:14]: Shouldn't be `3001` here, since that's the port we have in the env var?

**Review by wellfreire — COMMENTED** [review]
Apart from that port doubt, LGTM (as I have also tested this locally and it worked).

File: fixture.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash 
+
```
Comment (endeveit) [inline fixture.sh:2]: Could you add a little comment with a description of what this script does?

File: docker-compose.yml (hunk 2 above)
Comment (italolelis) [reply docker-compose.yml:14]: Guys this is an open source repo, we can't have this file here.... it shows the auth-service that is private

**Review by italolelis — CHANGES_REQUESTED** [review]
Guys lets review this, remember that this is an open source repo.....

**Comment by wellfreire** [comment]
@italolelis: You're right. I keep forgetting this is an open source thing. We need to start getting used to that =/

**Pull request closed without merging**